storage/rollup.config.js

```diff
@@ -0,0 +1,21 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
+
+export default {
+  input: 'dist/esm/index.js',
+  output: {
+    file: 'dist/plugin.js',
+    format: 'iife',
+    name: 'capacitorPlugin', // TODO: change this
```

Comment:
you should name the plugin and remove the TODO

Reply:
```suggestion
    name: 'capacitorStorage',
```